### PR TITLE
Workflow tests

### PR DESCRIPTION
**Pull request by aidan-wagner**
| | |
|:---|:---|
|**_Originating Project/Creator_**| |
|**_Affected Component_**|  |
|**_Affected Architectures(s)_**|  |
|**_Related Issue(s)_**|  |
|**_Has Unit Tests (y/n)_**|  |
|**_Builds Without Errors (y/n)_**|  |
|**_Unit Tests Pass (y/n)_**|  |
|**_Documentation Included (y/n)_**|  |

---
## Change Description

Added tests to workflow

## Rationale

Test code to ensure fprime-tools is not broken by future updates

## Future Work

Add tests for cookiecutter and backwards compatibility

### CODE REVIEW

**Comment by Joshua-Anderson** [comment]
@aidan-wagner Can you rebase this now that the cookiecutter changes have been merged in?

File: .github/workflows/fprime-tools-ci.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: fprime-tools test
+
+on: [pull_request]
```
Comment (Joshua-Anderson) [inline .github/workflows/fprime-tools-ci.yml]: We should set this up to be triggered on both pull requests and branch pushes.

**Review by Joshua-Anderson — COMMENTED** [review]
Other than the one comment I added looks good!

**Review by Joshua-Anderson — DISMISSED** [review]
LGTM

File: test/fprime/fbuild/test_build.py [hunk 2]
```diff
@@ -42,21 +42,6 @@ def get_data_dir():
     )
 
 
-def test_hash_finder():
```
Comment (LeStarch) [inline test/fprime/fbuild/test_build.py]: Why was this removed?  Should we not put in a test file and prove searching for hashes work?

**Review by LeStarch — COMMENTED** [review]
Just one question

**Review by Joshua-Anderson — DISMISSED** [review]

File: test/fprime/fbuild/test_build.py (hunk 2 above)
Comment (aidan-wagner) [reply test/fprime/fbuild/test_build.py]: I initially took it out because it needed a framework path, but I added a settings.ini with a fake framework path and was able to get it working!

**Review by LeStarch — APPROVED** [review]
